packages/core/addon/services/visualization.ts

```diff
@@ -7,14 +7,94 @@
  */
 
 import Service from '@ember/service';
+import type StoreService from '@ember-data/store';
 import config from 'ember-get-config';
 import { assert } from '@ember/debug';
 import { getOwner, setOwner } from '@ember/application';
-import type RequestFragment from 'navi-core/models/request';
 import { YavinVisualizationManifest } from 'navi-core/visualization/manifest';
+import NaviVisualizationConfigWrapper from 'navi-core/components/navi-visualization-config/wrapper';
+//@ts-ignore
+import ReportVisualization from 'navi-reports/components/report-visualization';
+import type RequestFragment from 'navi-core/models/request';
+import type NaviVisualizationBaseManifest from 'navi-core/navi-visualization-manifests/base';
+import type { TypedVisualizationFragment } from 'navi-core/models/visualization';
+import type { VisualizationType } from 'navi-core/models/registry';
+import type NaviFactResponse from 'navi-data/models/navi-fact-response';
+import type YavinVisualizationModel from 'navi-core/models/visualization-v2';
 
 const global = window as typeof window & { requirejs: { entries: Record<string, unknown> } };
 
+class CompatManifest extends YavinVisualizationManifest {
+  old: NaviVisualizationBaseManifest;
+
+  currentVersion;
+  namespace;
+  type;
+  niceName;
+  icon;
```

Comment:
Since this extends do we need this?

Reply:
Yea, it extends from the abstract base class so without it it fails
```
Non-abstract class 'CompatManifest' does not implement inherited abstract member 'icon' from class 'YavinVisualizationManifest<unknown>'.
```